### ISSUE TITLE
completed-docs does not report any errors

### ISSUE THREAD

**Issue by adibbleWLT** [palantir/tslint#2648]
### Bug Report

- __TSLint version__: 4.5.0
- __TypeScript version__: 2.2.2
- __Running TSLint via__:  CLI with "ng lint --type-check"

#### TypeScript code being linted

```ts
public anyFunctionWithoutDocumentation() {
    console.log('This function should throw a completed-docs linting error');
}
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "completed-docs": [ true, "classes", "methods", "functions" ]
    }
}
```

#### Actual behavior

No linting errors

#### Expected behavior

Should throw error: 
Methods must have documentation.

#### Explanation

After upgrading to Angular 4 this one rule does not get respected. All other lint settings (as far as currently noted) run as expected. Setting this rule as: "completed-docs": [ true, "classes", "methods", "functions" ], or even "completed-docs": true, does not cause a lint error even when I delete all documentation comments from all classes/methods/functions in a file. The file is definitely being linted as other rules do get flagged if they fail there. Linting configuration settings did not change in the upgrade process.

Note, I saw this ticket [#2291](https://github.com/palantir/tslint/issues/2291), but that uses Angular 2 and the the resolution does not apply here as --project tsconfig.json is no longer passed to the lint command (and trying to do so gets an error that "The option '--project' is not registered with the lint command."). Instead it is passed in the "lint" section of the .angular-cli.json file. This is correctly set however, and I can change settings in the tsconfig file to affirm that the changes are being reflected in the linting output.

**Comment by mbayou** [comment]
+1

**Comment by adibbleWLT** [comment]
Note: The latest upgrade 4.1.1 appears to have resolved this (and other later-noted) issues.
